client/assets/js/profile.js

```diff
@@ -0,0 +1,43 @@
+(function ready(fn) {
+  if (document.attachEvent ? document.readyState === "complete" : document.readyState !== "loading"){
```

Comment:
Using a ternary in this way is really confusing. Why is it being used inside an if declaration? The way I am reading this is that you are seeing if document.attachEvent is true. If so, check the truthyness of readyState is equal to "complete"? Otherwise check the unequalness of readystate not eqaul to loading? Then run the function that is sent into the ready function as an argument? There has to be a less jarring way of writing this.

Reply:
I asked Alicia to read it as well, and neither of us can understand what it is doing or why it is written in this way. So, Im not saying it is wrong or needs to be changed....but if it is going to be kept, I'd like to understand what is happening....